tests/utils/govc/cmd.go

```diff
@@ -50,3 +51,17 @@ func PowerOnVM(vmName string) string {
 	cmd := govc.PowerOnVM + vmName
 	return ssh.InvokeCommandLocally(cmd)
 }
+
+// GetDatastoreList returns a list of datastore names available
+func GetDatastoreList() []string {
+	log.Printf("Finding Datastores available on ESX")
+	cmd := govc.DatastoreInfo + govc.JSONTypeOutput + "| " + govc.JSONParser + govc.DatastoreList
+	out := ssh.InvokeCommandLocally(cmd)
+	return strings.Fields(out)
+}
+
+// GetVsanDatastore returns the name of vsan type datastore
+func GetVsanDatastore() string {
+	cmd := govc.DatastoreInfo + govc.JSONTypeOutput + "| " + govc.JSONParser + " '.Datastores[].Summary | select(.Type==\"vsan\") | .Name'"
+	return ssh.InvokeCommandLocally(cmd)
+}
```

Comment:
Please include this in the PR for just the common code and not part of this test.

Why do we need only a get() for VSAN DS